swatch-contracts/src/test/java/com/redhat/swatch/contract/CapacityReconciliationIntegrationTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.swatch.contract;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.redhat.swatch.contract.model.ReconcileCapacityByOfferingTask;
+import com.redhat.swatch.contract.repository.OfferingEntity;
+import com.redhat.swatch.contract.repository.OfferingRepository;
+import com.redhat.swatch.contract.repository.SubscriptionEntity;
+import com.redhat.swatch.contract.repository.SubscriptionMeasurementEntity;
+import com.redhat.swatch.contract.repository.SubscriptionRepository;
+import com.redhat.swatch.contract.service.CapacityReconciliationService;
+import io.quarkus.kafka.client.serialization.ObjectMapperSerde;
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+import io.quarkus.test.kafka.InjectKafkaCompanion;
+import io.quarkus.test.kafka.KafkaCompanionResource;
+import io.smallrye.reactive.messaging.kafka.companion.KafkaCompanion;
+import jakarta.inject.Inject;
+import jakarta.transaction.Transactional;
+import java.time.Duration;
+import java.time.OffsetDateTime;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+@Disabled("Running kafka w/ kubedock broken, see https://github.com/joyrex2001/kubedock/issues/53")
+@QuarkusTest
+@QuarkusTestResource(value = KafkaCompanionResource.class, restrictToAnnotatedClass = true)
+class CapacityReconciliationIntegrationTest {
+  @InjectKafkaCompanion KafkaCompanion companion;
```

Comment:
I know this issue cannot be reproduced using the smallrye in-memory component, so I like the idea of keeping this test here, but disabled, just to troubleshooting purposes.